amendments.json

```diff
@@ -520,5 +520,14 @@
       "status": "candidate",
       "id": 36
     }
+  ],
+  "closing-procedure": [
+    {
+      "description": "Prevent GC of non-closed RTCDataChannels",
```

Comment:
non-closing/non-closed

Reply:
Given where we're removing the channel I'm not sure we're preventing GC of non-closing channels?